ansible/bootstrap.yml

```
@@ -86,31 +90,13 @@
 
     - name: Run the nomad acl bootstrap process
       include_tasks: nomad_acl_bootstrap.yml
-      when: "{{ lookup('env', 'TF_VAR_nomad_acl') }}"
+      when: lookup('env', 'nomad_acl') | bool
 
     - name: Create roles for nomad in vault
       include_tasks: vault_nomad_integration.yml
```

```suggestion
      include_tasks: vault_nomad_integration.yml
      when: "{{ lookup('env', 'TF_VAR_nomad_acl') }}"
```
It makes no sense to run a vault - nomad integration without ACLs enabled in nomad. WIthout ACLs enabled there are no policies in nomad to create roles for etc..

---

The same statement applies on `consul secrets engine`